### PR TITLE
Fix snowflakes

### PR DESCRIPTION
**Pull request by X1nto**
Because of how fucking stupid Discord is, some users reported that sometimes snowflakes contain negative values (which, should NEVER happen). Using ULongs crashes the HTTP client as it does not expect a negative value when serializing.

### CODE REVIEW

File: app/src/main/java/com/xinto/opencord/domain/manager/PersistentDataManager.kt [hunk 1]
```diff
@@ -4,52 +4,52 @@ import android.content.SharedPreferences
...
     }
 
     companion object {
         const val CURRENT_GUILD_ID_KEY = "current_guild_id"
-        const val CURRENT_GUILD_ID_DEFAULT = 0L
+        const val CURRENT_GUILD_ID_DEFALT = 0L
 
         const val CURRENT_CHANNEL_ID_KEY = "current_channel_id"
-        const val CURRENT_CHANNEL_ID_DEFAULT = 0L
+        const val CURRENT_CHANNEL_ID_DEFALT = 0L
```
Comment (DiamondMiner88) [inline app/src/main/java/com/xinto/opencord/domain/manager/PersistentDataManager.kt:52]: Looks like you did a faulty search and replace here for UL

**Review by DiamondMiner88 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/xinto/opencord/domain/manager/PersistentDataManager.kt (hunk 1 above)
Comment (wingio) [reply app/src/main/java/com/xinto/opencord/domain/manager/PersistentDataManager.kt:52]: i like this spelling better

Comment (X1nto) [reply app/src/main/java/com/xinto/opencord/domain/manager/PersistentDataManager.kt:52]: AHAHAHAHHAH LMAO

**Comment by DiamondMiner88** [comment]
afaik the negative ids happen on guild folders only

**Comment by X1nto** [comment]
> afaik the negative ids happen on guild folders only

Honestly, still, no reason to really keep ULongs. The Snowflake max size is Long.MAX_VALUE anyway, so it's not like we're losing numbers when using signed instead of unsigned

**Review by DiamondMiner88 — APPROVED** [review]

**Pull request merged**